src/rmt.rs

```diff
@@ -11,7 +11,7 @@
 //!
 //! Not supported:
 //! * Interrupts.
-//! * Receiving.
+//! * ~Receiving.~
```

Comment:
This comment is outdated. I should also add something about how the legacy rmt is incompatible with the onewire rmt.